src/test/scala/gatling/ReviewingAPie.scala

```diff
@@ -0,0 +1,62 @@
+package gatling
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import scala.util.Random
+
+object ReviewingAPie {
+  private val searchData = Array(
+    Map("search" -> "pie")
+  )
+  private val reviewsData = Array(
```

Comment:
Instead of using Map array for inserting review and search data use a csv feeder. (Download google sheets csv file and add it to resources/gatling/data).